Assets/scss/components/_masonry-layout.scss

```diff
@@ -0,0 +1,66 @@
+.cmp-masonry-layout {
+  background-color: var(--gray92);
+
+  &__content {
+    max-width: toRem(1200);
+    margin: 0 auto;
+    column-count: 1;
+
+    @include min-screen(sm) {
+      column-count: 3;
```

Comment:
`column-count` is underused, great for this design!

I know we don't have a tablet design, but you could switch to 2 cols at `sm` then 3 cols at `md` or `lg`. Not mandatory, just good to think about all breakpoints even when we only have one or two designs.

Reply:
@jwetmore-hoodoo - I have updated the tablet design.